src/notes/noteHelpers.ts

```diff
@@ -152,7 +152,8 @@ export async function findNote(config: Map<ConfigurationKeys, string>) {
             name: "filePath",
             excludePath: (nodePath: string) =>
                 nodePath.startsWith("node_modules"),
-            excludeFilter: (nodePath: string) => nodePath.startsWith("."),
+            excludeFilter: (nodePath: string) =>
+                nodePath.match(/.contents.md/gm) !== null,
```

Comment:
Instead of singling out `.contents`, can we exclude _all_ `.` files (I like that approach better honestly).

One way to do that would be:
```suggestion
            excludeFilter: (nodePath: string) =>
                nodePath.startsWith(`${rootDir}/.`),
```

Reply:
I also moved a bunch of stuff -- so if it's easier for me to just bring this into a different branch, I can.